src/article.ts

```diff
@@ -0,0 +1,79 @@
+// ----- Imports ----- //
+
+import { Pillar, pillarFromString } from 'pillar';
+import { Content } from 'capiThriftModels';
+import { isFeature, isAnalysis, isImmersive, isReview } from 'capi';
+
+
+// ----- Types ----- //
+
+const enum Layout {
+    Default,
+    Immersive,
+    Feature,
+    Review,
+    Analysis,
+    Opinion,
+    Liveblog,
+    Gallery,
+    Interactive,
+    Picture,
```

Comment:
Is this `PhotoEssay`?

Reply:
This came from the [`ContentType` enum](https://github.com/guardian/apps-rendering/blob/master/src/capiThriftModels.ts#L26). Assuming it's the same thing, I don't mind too much what it's called. If it's *not* the same thing then we may need a `PhotoEssay` layout.

Reply:
It sounds like it's the same thing so makes sense to keep the capi type.